pkg/utils/read_write_layer/logger.go

```diff
@@ -1,269 +1,273 @@
 package read_write_layer
 
 import (
+	"context"
+	"fmt"
+
 	lsv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
 	lsv1alpha1helper "github.com/gardener/landscaper/apis/core/v1alpha1/helper"
+	"github.com/gardener/landscaper/controller-utils/pkg/logging"
+	lc "github.com/gardener/landscaper/controller-utils/pkg/logging/constants"
 	"github.com/gardener/landscaper/pkg/utils"
 )
 
 const (
-	historyLogLevel       = 1
-	keyName               = "name"
-	keyNamespace          = "namespace"
-	keyPhase              = "phase"
-	keyPhaseDeployer      = "phase-d"
-	keyWriteID            = "write-id"
-	keyGenerationOld      = "gen-old"
-	keyGenerationNew      = "gen-new"
-	keyOperation          = "op"
-	keyResourceVersionOld = "rv-old"
-	keyResourceVersionNew = "rv-new"
-	keyJobID              = "j-id"
-	keyJobIDFinished      = "j-id-f"
+	historyLogLevel          logging.LogLevel = logging.DEBUG
```

Comment:
I would recommend to set this on log level INFO, because this is the only possibility to analyze errors in the central reconcile logic.